manifests/init.pp

```diff
@@ -82,6 +83,12 @@
     $include_libsudo_vas_real = $include_libsudo_vas
   }
 
+  if is_string($hiera_merge_sudoers) {
+    $hiera_merge_sudoers_bool = str2bool($hiera_merge_sudoers)
+  } else {
+    $hiera_merge_sudoers_bool = $hiera_merge_sudoers
+  }
+
```

Comment:
This is needed or you could pass a bad type
`validate_bool($hiera_merge_sudoers_bool)`